packages/js-client/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "@aragon/sdk-client-template",
+  "author": "Aragon Association",
+  "version": "1.0.0",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "engines": {
+    "node": ">=16"
+  },
+  "scripts": {
+    "start": "tsdx watch",
+    "build": "tsdx build",
+    "test": "TESTING=true tsdx test --verbose --runInBand",
+    "lint": "tsdx lint",
+    "prepare": "tsdx build",
+    "size": "size-limit",
```

Comment:
```
➜  js-client git:(f/OS-465-dummy-js-client) yarn size
yarn run v1.22.19
$ size-limit
✔ Adding to empty esbuild project
  
  
  Size Limit can’t find files at dist/plugin-client.cjs.production.min.js
  
  Size Limit can’t find files at dist/plugin-client.esm.js
error Command failed with exit code 1.
```

Reply:
Is the name of the artifacts the same? I assume you ran the build command

Reply:
this is solved